plugins/net_plugin/net_plugin.cpp

```diff
@@ -3927,16 +3975,24 @@ namespace eosio {
    void net_plugin::set_program_options( options_description& /*cli*/, options_description& cfg )
    {
       cfg.add_options()
-         ( "p2p-listen-endpoint", bpo::value< vector<string> >()->default_value( vector<string>(1, string("0.0.0.0:9876")) ), "The actual host:port used to listen for incoming p2p connections. May be used multiple times.")
+         ( "p2p-listen-endpoint", bpo::value< vector<string> >()->default_value( vector<string>(1, string("0.0.0.0:9876:0")) ), "The actual host:port used to listen for incoming p2p connections. May be used multiple times. "
+           "Block syncing to all peers connected via the port will be throttled to the specified rate. "
+           "See the 'p2p-peer-address' argument for format details.")
          ( "p2p-server-address", bpo::value< vector<string> >(), "An externally accessible host:port for identifying this node. Defaults to p2p-listen-endpoint. May be used as many times as p2p-listen-endpoint. If provided, the first address will be used in handshakes with other nodes. Otherwise the default is used.")
          ( "p2p-peer-address", bpo::value< vector<string> >()->composing(),
            "The public endpoint of a peer node to connect to. Use multiple p2p-peer-address options as needed to compose a network.\n"
-           "  Syntax: host:port[:<trx>|<blk>]\n"
+           "  Syntax: host:port[:<trx>|<blk>][:<rate-cap>]\n"
            "  The optional 'trx' and 'blk' indicates to node that only transactions 'trx' or blocks 'blk' should be sent."
+           "  The optional rate cap will limit block sync bandwidth to the specified rate.  A number alone will be "
+           "  interpreted as bytes per second.  The number may be suffixed with units.  Supported units are: "
```

Comment:
Unless I'm looking at this wrong a number alone is ignored.

Reply:
It wasn't meant to be.  Unspecified units was meant to allow `:0` as shorthand for explicitly unthrottled.  I got paranoid about `match[1]` but as it turns out if the match count is 1, that index returns an empty string, which then corresponds to the first entry in the `prefix_multipliers` table which multiplies by 1 if units are empty.  Removed check of `count()`.